tests/testthat/_snaps/cassette_class.md

```diff
@@ -100,3 +100,32 @@
       Error in `initialize()`:
       ! `name` must not be the same as an existing cassette.
 
+# important interactions are logged
+
+    Code
+      use_cassette("test", httr::GET(hb("/get")))
```

Comment:
@sckott I think this is the best place to look at. Do you see all the information you'd want in the log?

Reply:
```r
[Cassette: "x"] Handling request: GET http://example.com application/json, text/xml, application/xml, */* aefadsf 5
[Cassette: "x"]   looking for existing requests using uri/headers
[Cassette: "x"]   no matching requests
[Cassette: "x"] - unhandled
[Cassette: "x"]   no match: GET http://example.com
```

I assume we want to remove that `-` prefixing `unhandled`?

Reply:
otherwise looks great

Reply:
How do you get that log?

Reply:
```r
library(httr)
vcr::local_vcr_configure_log()

vcr::local_cassette("test2", match_requests_on = c("method", "uri", "headers"))
GET("http://example.com")
vcr::eject_cassette()

vcr::local_cassette("test2", match_requests_on = c("method", "uri", "headers"))
GET("http://example.com", add_headers(A = 5))
vcr::eject_cassette()
```